app/assets/javascripts/modules/holiday-types.es6

```diff
@@ -0,0 +1,39 @@
+/* global TapBase */
+{
+  'use strict';
+
+  class HolidayTypes extends TapBase {
```

Comment:
Got a few thoughts on this bit but we can revisit when we take a look at the UI stuff for this page? cc @Guntrisoft